tools/accuracy_checker/configs/face-detection-0105.yml

```diff
@@ -0,0 +1,48 @@
+models:
+  - name: face-detection-0105
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   intel/face-detection-0105/FP32/face-detection-0105.xml
+        weights: intel/face-detection-0105/FP32/face-detection-0105.bin
+        adapter:
+          type: class_agnostic_detection
```

Comment:
you forgot replace adapter or output is not changed for this model?

Reply:
the same question about 0106

Reply:
no, i did not forget. models 0105 and 0106 should use class_agnostic_detection adapter

p.s. I checked their outputs, they are `boxes` and `labels` (as expected)

Reply:
ok, I understand